src/main/app/fileUpload/documentManager.ts

```diff
@@ -2,16 +2,19 @@
 
 import axios, { AxiosInstance, RawAxiosRequestHeaders } from 'axios';
 import config from 'config';
+import { Logger } from '../../../test/unit/mocks/hmcts/nodejs-logging';
 
 // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
 export const uploadDocument = async (formData, s2sToken, req) => {
+  const logger = Logger.getLogger('documentManager');
```

Comment:
move this out of the function so it can be used everywhere if this script was to grow